src/orchestrator/atom.ts

```diff
@@ -33,14 +33,14 @@ export class Atom {
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
   over(lens: any, fn: any) {
     //eslint-disable-next-line @typescript-eslint/no-explicit-any
-    this.update(R.over(lens, fn) as any as (OrchestratorState) => OrchestratorState);
+    this.update(R.over(lens, fn) as any as (state: OrchestratorState) => OrchestratorState);
```

Comment:
The linter wanted to bind the parameter. I don't know if this is a newer version of TS thing or not.